templates/shared/_server-download-newsletter.html

```diff
@@ -0,0 +1,35 @@
+<div class="p-section--shallow">
```

Comment:
Tabbing for this file looks weird, but might just be a gh thing

Reply:
Should be fixed now. I think I finally know what's going on.